tests/Functional/Execution/ValuesResolverTest.php

```diff
@@ -58,7 +49,7 @@ public function testGetValues()
             $schema, [], null, null, ['name' => 'Han Solo'], null, $operation, []
         );
 
-        $args = $this->valuesResolver->coerceArgumentValues($definition, $node, $context->getVariableValues());
+        $args = getArgumentValues($definition, $node, $context->getVariableValues());
```

Comment:
I just think `getArgumentValues` is not the right name for the function, because values is really got from `$context->getVariablesValues`

Reply:
That's what they call it in the reference implementation so I'd prefer to keep it like that for now.

Reply:
Actually I commented on this before, in the graphql specification they call it `coerceArgumentValues`

Reply:
I can change this, should I also change the associated directive-method?